### ISSUE TITLE
Downloading particular chapters rather than entire course

### ISSUE THREAD

**Issue by Sushant199** [udacimak/udacimak#141]
I have to download many courses and extracurricular section of many courses is roughly same. It's a waste of expensive mobile internet data balance to download all of them.

Is there a option from which only desired chapters can be downloaded?

**Comment by udacimak** [comment]
I didn't create a feature to only download some part of the course.

If you'd like to do that, editing the original JSON files could be one option.

**Comment by Sushant199** [comment]
I can just delete the folders of the chapters in the json directory. Would that be okay?
Would it conflict with other things or create problems in structure of the whole course?

**Comment by udacimak** [comment]
Yes, technically you can do that because the `render` command goes through each JSON file in the downloaded directory to render the content. So if some directories are deleted, Udacimak will skip those. Let me know if you have any issues
